src/main/java/org/beanpod/switchboard/controller/EncoderController.java

```diff
@@ -78,19 +86,25 @@
     if (encoderDto.getOutput().isEmpty()) {
```

Comment:
This also returns the users' password hash.

Reply:
I'm glad you caught this. It's a side-effect I noticed as well. I wasn't too sure how to fix this. The fact that some controllers used Models and others used DTOs was something I brought up earlier in the project. I was not sure if one was better than the other, so I avoided changing things myself. Thanks to your review comments, we see that Models are the way to go as it allows us to present subset responses. I feel that a conversion effort should be done to go from DTO to Model, and again this should have probably been caught in an earlier review.